package.json

```diff
@@ -35,21 +34,20 @@
   "bugs": "https://github.com/okta/samples-js-vue/issues",
   "homepage": "https://github.com/okta/samples-js-vue",
   "devDependencies": {
-    "axios": "^1.6.4",
-    "dotenv": "^8.6.0",
+    "axios": "^1.7.9",
+    "dotenv": "^16.4.7",
```

Comment:
:whistle: that's a big bump. dotenv's surface area is relatively small, but you've checked this doesn't break?